src/shared/components/Banners/TransferOwnershipSuccessBanner/TransferOwnershipSuccessBanner.tsx

```diff
@@ -0,0 +1,22 @@
+import { Trans } from 'react-i18next';
+
+import { Banner, BannerProps } from '../Banner';
+
+export const TransferOwnershipSuccessBanner = ({ children, email, ...props }: BannerProps) => (
+  <Banner
+    duration={7000}
```

Comment:
Could you please create a variable for the duration and place it in the TransferOwnershipSuccessBanner.const.ts file?